quipucords/fingerprinter/runner.py

```diff
@@ -1038,43 +1032,59 @@ def _process_network_fact(self, source: dict, fact: dict) -> dict:
                     else "ip_address_show_ipv4"
                 ),
                 "ip_addresses",
+                formatters.list_or_none,
```

Comment:
That one also is a `JSONField`... so perhaps `installed_products` should be formatted to list or dict instead of string?